content/en/docs/setup/container.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Deploy as a container
+description: Learn how to run and manage Falco as a container
+slug: container
+aliases:
+- ../install-operate/running
+- ../getting-started/running
+weight: 20
+---
+
+{{% pageinfo color="primary" %}}
+Falco consumes streams of events and evaluates them against a set of security {{< glossary_tooltip text="rules" term_id="rules" >}} to detect abnormal behavior. By default, Falco is preconfigured to consume events from the Linux Kernel. This scenario requires Falco to be privileged, and depending on the kernel version installed on the node, a {{< glossary_tooltip text="driver" term_id="drivers" >}} will be installed on the node. Since orchestration systems like Kubernetes are out of scope for this section, it's up to the user to manage the container lifecycle and deployment across the nodes.
+
+For other installation scenarios, such as consuming cloud events or other data sources using plugins, please refer to the [Plugins](docs/plugins/) section.
+{{% /pageinfo %}}
+
+## Install
+
+This section describes how to run the Falco userspace process in a container using one of the released [container images](/docs/download#images).
+
+By default, Falco is preconfigured to consume events from the Linux Kernel. For this default installation scenario, Falco can be run in two ways:
+
+- [Fully Privileged](#docker-privileged)
+- [Least Privileged (Recommended)](#docker-least-privileged)
+
+Different instructions apply to each method depending on the driver used. Note that the **{{< glossary_tooltip text="Modern eBPF" term_id="modern-ebpf-probe" >}} does not require driver installation**.
+
+### Fully Privileged {#docker-privileged}
+
+To run Falco in a container using Docker with full privileges, use the following commands:
+
+#### Modern eBPF {#docker-privileged-modern-ebpf}
+
+The {{< glossary_tooltip text="Modern eBPF" term_id="modern-ebpf-probe" >}} is bundled into the Falco binary. Therefore, the `falcosecurity/falco-no-driver` image is enough to run Falco. This allows you to run Falco without dependencies by using the following command:
+
+```shell
+docker pull falcosecurity/falco-no-driver:latest
+docker run --rm -it \
+           --privileged \
+           -v /var/run/docker.sock:/host/var/run/docker.sock \
+           -v /proc:/host/proc:ro \
+           -v /etc:/host/etc:ro \
+           falcosecurity/falco-no-driver:latest
+```
+
+#### eBPF Probe {#docker-privileged-ebpf}
```

Comment:
Why modern -> ebpf -> kmod instead of modern -> kmod -> ebpf ? I think we had this discussion, but let's put it in the PR here so we don't forget.

Reply:
I changed my mind a couple of times regarding this. Although I agree we use a different ordering when the automatic selection is enabled, it may be easier for the reader to have both eBPF solutions very close since they have some similarities. Moreover, from an installation point of view, the legacy eBPF sits precisely in the middle between the modern and the kmod.

That said, I don't have strong opinions or compelling motivations. 

I'm ok to switch the order. Before doing that, I just summon @FedeDP since he worked a lot on the automatic selection, and may have opinions.

Reply:
I'd use the same priority-list we used for driver types in Falco, that is: 
* modern_ebpf
* kmod
* ebpf

See https://github.com/falcosecurity/falco/blob/master/scripts/CMakeLists.txt#L51

Reply:
ok two vs. one, you won :smile_cat: 

Changing the ordering.